nuget/Microsoft.Windows.CsWinRT.targets

```diff
@@ -67,12 +67,30 @@ Copyright (C) Microsoft Corporation. All rights reserved.
     </PropertyGroup>
   </Target>
 
+  <!-- 
+      During a design-time build, input WinMDs may not exist yet if a full build has not been completed.
+      If that is the case, generation of the CsWinRT projection will fail. To avoid causing a design-time
+      build break (which can put the design-time build results of dependent projects into a bad state),
+      we want to skip projection generation if a) this is a design-time build, and b) any of the inputs
+      do not exist.
+  -->
+  <Target Name="CsWinRTCalculateSkipDesignTimeGenerateProjection" 
+          DependsOnTargets="CsWinRTRemoveWinMDReferences">
+    <ItemGroup>
+      <_CsWinRTNonexistentInputs Include="@(CsWinRTInputs)" Condition="!Exists('%(CsWinRTInputs.Identity)')" />
+    </ItemGroup>
+
+    <PropertyGroup>
+      <CsWinRTSkipDesignTimeGenerateProjection Condition="('$(DesignTimeBuild)' == 'true' OR '$(BuildingProject)' != 'true') and '@(_CsWinRTNonexistentInputs)'!=''">true</CsWinRTSkipDesignTimeGenerateProjection>
```

Comment:
>CsWinRTSkipDesignTimeGenerateProjection [](http://example.com/codeflow?start=7&length=39)

Ideally we'd just override `$(CsWinRTGenerateProjection)` and not have to change the `CsWinRTGenerateProjection` target at all, but unfortunately MSBuild appears to evaluate target `Condition`s *before* any target gets a chance to run.